src/test/kotlin/com/zepben/evolve/services/network/NetworkServiceTest.kt

```diff
@@ -100,108 +100,129 @@ internal class NetworkServiceTest {
     @Test
     internal fun testNetworkConnect() {
         val network = NetworkService()
-        val node0 = createNodeForConnecting(network, "node0", 2)
-        val node1 = createNodeForConnecting(network, "node1", 2)
-        val node2 = createNodeForConnecting(network, "node2", 2)
-        val node3 = createNodeForConnecting(network, "node3", 2)
-        val node4 = createNodeForConnecting(network, "node4", 2)
-        val node5 = createNodeForConnecting(network, "node5", 2)
+        val j0 = createJunctionForConnecting(network, "j0", 2)
```

Comment:
I found it a bit confusing to name junctions node0/node1/etc since we also have connectivity nodes in a node breaker model, so I renamed all of them to j0/j1/etc. I left `StupidlyLargeNetwork` untouched as per the contribution doc.